org.naev.Naev.json

```diff
@@ -45,7 +45,6 @@
     {
       "name": "naev",
       "config-opts": [
-        "--disable-debug",
```

Comment:
Again why for this? Why would people want to download a debug build?

Reply:
It's not debug build just because of debug symbols presence. These are always useful to obtain more useful stacktrace or run gdb from Sdk and comes for free as they are extracted to separate extension that is not installed by default.

Reply:
It is a debug build though. When you run the app it will tell you what kind of build it is. Without this flag it will build as debug. Unless you mean there's some way flatpak-builder creates two versions, a debug and a release build?